po/es.po

```diff
@@ -81,158 +80,150 @@ msgstr "_Cancelar"
 #: src/accountdialog.ui:22 src/remotedesktopdialog.ui:28
 #: src/screencastdialog.ui:28 src/screenshotdialog.ui:38
 msgid "_Share"
-msgstr "_Compartir"
+msgstr "Co_mpartir"
 
 #: src/accountdialog.ui:148
 msgid "Make changes before sharing the information"
-msgstr ""
+msgstr "Realizar cambios antes de compartir la información"
 
 #: src/appchooserdialog.c:154
 msgid "Failed to start Software"
-msgstr ""
+msgstr "No se pudo iniciar Software"
 
 #: src/appchooserdialog.c:373
 #, c-format
 msgid ""
 "Select an application to open “%s”. More applications are available in <a "
 "href='software'>Software.</a>"
 msgstr ""
+"Seleccione una aplicación para abrir «%s». Hay más aplicaciones "
+"disponibles en <a href='software'>Software</a>."
 
 #: src/appchooserdialog.c:378
 msgid ""
 "Select an application. More applications are available in <a "
 "href='software'>Software.</a>"
 msgstr ""
+"Seleccione una aplicación. Hay más aplicaciones disponibles en <a "
+"href='software'>Software</a>."
 
 #: src/appchooserdialog.c:391
 #, c-format
 msgid "Unable to find an application that is able to open “%s”."
-msgstr ""
+msgstr "No se encontró ninguna aplicación capaz de abrir «%s»."
 
 #: src/appchooserdialog.c:396
 msgid "Unable to find a suitable application."
-msgstr ""
+msgstr "No se encontró ninguna aplicación adecuada."
 
 #: src/appchooserdialog.ui:13
 msgid "Open With"
-msgstr "Abrir Con"
+msgstr "Abrir con"
```

Comment:
Header capitalization ("con" is a preposition, but it's also the last word of the sentence, so it should be capitalized)